### PR TITLE
Fix tests to make sure they are passing

### PR DESCRIPTION
**Pull request by mshirlaw**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (mattseddon) [inline babel.config.js:1]: [C] I'd prefer to go with typescript off the bat

File: package.json [hunk 2]
```diff
@@ -10,14 +10,20 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "jest --notify --config=jest.config.json"
+    "test": "jest --notify --config=jest.config.js"
```
Comment (mattseddon) [inline package.json:13]: [Q] Do you need to declare a config isn't it auto-magically found?

**Review by mattseddon — APPROVED** [review]
LGTM

File: babel.config.js (hunk 1 above)
Comment (mshirlaw) [reply babel.config.js:1]: Thanks will update

File: package.json (hunk 2 above)
Comment (mshirlaw) [reply package.json:13]: Probably would be, will confirm

**Pull request merged**